frontstage_api/error_handlers.py

```diff
@@ -18,11 +18,15 @@ def api_error_method(error):
         "error": {
             "url": error.url,
             "status_code": error.status_code,
-            "data": error.data
+            "data": error.data,
+            "description": error.description
         }
     }
     status_code = error.status_code if error.status_code else 500
-    logger.error('Error during api call', url=error.url, status_code=error.status_code)
+    if status_code == 500:
+        logger.error(error.description, url=error.url, status_code=error.status_code, **error.kwargs)
```

Comment:
Is there reasoning behind the error kwargs being logged for this error but not all?

Reply:
The ApiError is the only one that has optional kwargs to log.

Reply:
`status_code` should be logged as `status` according to SDC logging standards